shanoir-ng-users/src/main/java/org/shanoir/ng/email/EmailServiceImpl.java

```diff
@@ -417,20 +425,38 @@ public void notifyStudyManagerDataImported(DatasetImportEmail generatedMail) {
 		}
 	}
 
-	private class DatasetDetail {
-		private String url;
-		private String name;
-		public String getUrl() {
-			return url;
-		}
-		public void setUrl(String url) {
-			this.url = url;
-		}
-		public String getName() {
-			return name;
-		}
-		public void setName(String name) {
-			this.name = name;
+	@Override
+	public void notifyStudyManagerStudyUsersAdded(EmailStudyUsersAdded email) {
+        // Find user that edited the study
+        User user = userRepository.findById(email.getUserId()).orElse(null);
+
+		// Get the list of recipients
+		List<User> studyAdmins = (List<User>) this.userRepository.findAllById(email.getRecipients());
+		
+		List<User> newStudyUsers = this.userRepository.findByIdIn(email.getStudyUsers());
+		
+		for (User studyAdmin : studyAdmins) {
+			MimeMessagePreparator messagePreparator = mimeMessage -> {
+				final MimeMessageHelper messageHelper = new MimeMessageHelper(mimeMessage);
+				messageHelper.setFrom(administratorEmail);
+				messageHelper.setCc(user.getEmail());
+				messageHelper.setTo(studyAdmin.getEmail());
+				messageHelper.setSubject("[Shanoir] Member(s) added to " + email.getStudyName());
+				final Map<String, Object> variables = new HashMap<>();
+				variables.put(FIRSTNAME, studyAdmin.getFirstName());
+				variables.put(LASTNAME, studyAdmin.getLastName());
+				variables.put(EMAIL, user.getEmail());
+				variables.put(STUDY_NAME, email.getStudyName());
+				variables.put(STUDY_USERS, newStudyUsers);
+				variables.put(SERVER_ADDRESS, shanoirServerAddress + "study/edit/" + email.getStudyId());
+				final String content = build("notifyStudyAdminStudyUsersAdded", variables);
+				LOG.error(content);
```

Comment:
- ` LOG.error(content);`
should not be here (but also upper in my code..)

Reply:
done, changed to info